src/templates/PoolManager/ActivityCard/Swap/styles.ts

```diff
@@ -0,0 +1,27 @@
+import styled, { css } from 'styled-components'
+
+export const Swap = styled.ul`
+  ${() => css`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: 2.4rem;
+  `}
+`
```

Comment:
You're not using any variable from `styled-components`, this arrow function is unnecessary
```suggestion
export const Swap = styled.ul`
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  gap: 2.4rem;
`
```